tfhe-csprng/src/generators/aes_ctr/mod.rs

```diff
@@ -206,6 +206,7 @@ pub use index::*;
 
 /// A module containing structures to manage table indices and buffer pointers together properly.
 mod states;
+#[cfg(feature = "parallel")]
 pub use states::*;
```

Comment:
looks weird to me, should we instead remove this pub use and do the correct imports needed in parallel.rs ?